MovieApp/MovieApp/App/Features/MovieBrowser/MoviesViewController/HomePage2/HomePageTwoViewController.swift

```diff
@@ -7,11 +7,56 @@ class HomePageTwoViewController: UIViewController {
     var navigationView: MovieAppNavigationView!
     var searchBar: MovieSearchBar!
     var subcategoryView: SubcategoryView!
+    var moviesCollectionView: UICollectionView!
+    var flowLayout: UICollectionViewFlowLayout!
         
     override func viewDidLoad() {
         super.viewDidLoad()
         
         buildViews()
+        setupCollectionView()
+    }
+    
+    private func setupCollectionView() {
+        moviesCollectionView.register(NewMovieCell.self, forCellWithReuseIdentifier: NewMovieCell.reuseIdentifier)
+        moviesCollectionView.dataSource = self
+        moviesCollectionView.delegate = self
+    }
+    
+}
+
+extension HomePageTwoViewController: UICollectionViewDataSource {
+    
+    func collectionView(_ collectionView: UICollectionView, numberOfItemsInSection section: Int) -> Int {
+        return 10
+    }
+    
+    func collectionView(
+        _ collectionView: UICollectionView,
+        cellForItemAt indexPath: IndexPath
+    ) -> UICollectionViewCell {
+        guard let cell = collectionView.dequeueReusableCell(
+                withReuseIdentifier: NewMovieCell.reuseIdentifier,
+                for: indexPath) as? NewMovieCell
+        else {
+            return UICollectionViewCell()
+        }
+        let mockMovie = MovieViewModel(identifier: 13, title: "Mock", overview: "Bla bla bla", posterPath: URL(string: "https://image.tmdb.org/t/p/w185/qAZ0pzat24kLdO3o8ejmbLxyOac.jpg"))
+        
```

Comment:
Pushing mocked data is ok for now so that I can check this as well, but generally, the idea would be to create the PR screenshots with mocked data and not to push them. But at the same time, make it easy to just add an array of VM-s which would serve as mock data.